backend/app/views/spree/admin/users/_lifetime_stats.html.erb

```diff
@@ -8,19 +8,31 @@
     <table class="table" id="user-lifetime-stats" data-hook>
       <tr>
         <td width="30%"><%= Spree.t(:total_sales) %>:</td>
-        <td><%= @user.display_lifetime_value.to_html %></td>
+        <td>
+          <% current_store.supported_currencies.split(',').each do |currency| %>
+            <div><%= @user.display_lifetime_value(store: current_store, currency: currency).to_html %></div>
+          <% end %>
+        </td>
       </tr>
       <tr>
         <td><%= Spree.t(:num_orders) %>:</td>
-        <td><%= @user.order_count %></td>
+        <td><%= @user.order_count(current_store) %></td>
       </tr>
       <tr>
         <td><%= Spree.t(:average_order_value) %>:</td>
-        <td><%= @user.display_average_order_value.to_html %></td>
+        <td>
+          <% current_store.supported_currencies.split(',').each do |currency| %>
+            <div><%= @user.display_average_order_value(store: current_store, currency: currency).to_html %></div>
+          <% end %>
+        </td>
       </tr>
       <tr>
         <td><%= Spree.t('admin.user.store_credits') %>:</td>
-        <td id="store_credit"><%= Spree::Money.new(@user.total_available_store_credit) %></td>
+        <td id="store_credit">
+          <% current_store.supported_currencies.split(',').each do |currency| %>
+            <div><%= Spree::Money.new(@user.total_available_store_credit(currency, current_store), currency: currency) %></div>
```

Comment:
@tomurb let's add `display_total_available_store_credit` method so we won't call `Spree::Money` directly in views